thirtybees/merchants-guide/introduction.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Introduction
+tags:
+  - "Introduction"
+type: first_page
+homepage: true
+audience: docs
+permalink: /
```

Comment:
Permalink should be /merchants-guide/ I think. This URL is relative from the root of the site. Therefore `homepage` should also be set to `false`.